### PR TITLE
2024 Annual Report Hyperledger Aries

### PR DESCRIPTION
**Pull request by swcurran**
Signed-off-by: Stephen Curran <swcurran@gmail.com>

### CODE REVIEW

File: project-reports/2024/2024-annual-Hyperledger-Aries.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+Aries remains a healthy project and one of several viable candidates for major digital trust / verifiable credential deployments. Aries excels in its maturity, focus on privacy, and enabling a lasting trusted digital relationship between parties. Further, there have been multiple demonstrations this year that regardless of where the winds push Digital Trust, the Aries codebases will be a fit.
+
+2023 has been an interesting year in the Aries project. Aries Cloud Agent Python and Aries VCX continued to thrive, adding capabilities and supporting repositories (such as the Aries ACA-Py Plugins repo). A couple of smaller, long dormant sub-projects were archived (Aries Framework .NET and Aries Framework Go), and Agent Framework JavaScript moved to the Open Wallet Foundation. Agent Framework JavaScript remains interoperable and a close collaborator with the Aries frameworks, Aries Cloud Agent Python and Aries VCX. Current discussions are ongoing about transferring Aries Bifold to OWF as well, as it is so tied to Agent Framework JavaScript.
```
Comment (denyeart) [inline project-reports/2024/2024-annual-Hyperledger-Aries.md:20]: It looks like Aries Framework Go has not been archived. In fact, there are some dependencies on it and a proposal to keep some of the code alive, perhaps in a different repository.
For details see https://github.com/hyperledger/aries-framework-go/issues/3633.
Who could help talk to this issue and the possibility of keeping some of the Aries Framework Go alive?

**Review by arsulegai — APPROVED** [review]

File: project-reports/2024/2024-annual-Hyperledger-Aries.md (hunk 1 above)
Comment (swcurran) [reply project-reports/2024/2024-annual-Hyperledger-Aries.md:20]: Interesting. The primary developers of AFGo were primarily at Gen Digital, and they dropped the majority of their Digital Trust team, including (AFAIK) all of the developers that had been working on AFGo.  The challenge of having a repo dependent on one team.

As far as pulling out the BBS+ code as requested in the noted issue — I assume that can be done independent of the project, especially if the developers don’t respond.  It’s open source software :-). ACA-Py is using a different BBS+ implementation (the one from Matt Global), so this didn’t get raised to the broader community.

There is lots going on with BBS+ implementations, so I’m not sure what is the “right” one to be used in the Aries community. That issue will likely come up in the AnonCreds v2 work as we add BBS+ Signatures to the implementation.

File: project-reports/2024/2024-annual-Hyperledger-Aries.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+* Aries RFC Cleanup — Status Review and the human friendly publication of the Aries Protocol RFCs integrated with the DIDComm Protocols.
+* An evolution of the 'Start Here' documentation for the Aries Frameworks
+
+A backdrop to the activity in Aries in 2024 will be the ongoing evolution of the relationship between Aries and the Open Wallet Foundation. Given the connections between the participants in both efforts and the shared knowledge, we’re confident that there will continue to be good collaboration and a focus on interoperability across the projects.
+
+### Help Required
+
+_Guidance: How can the Hyperledger Foundation or the TOC help you achieve your upcoming goals?_
+
+Keeping open the dialogue about where the components of Aries should be between Hyperledger and the Open Wallet Foundation. From a bottom up perspective, there is a diversity of opinions on that question (as indicated by the Credo/Agent Framework JavaScript and Bifold moves), and we’re very interested in the top down perspective. Likewise, the relationship between Aries and DIF concerning DIDComm is another interesting “challenge”.
```
Comment (TelegramSam) [inline project-reports/2024/2024-annual-Hyperledger-Aries.md]:
```suggestion
Keeping open the dialogue about where the components of Aries should be between Hyperledger and the Open Wallet Foundation. From a bottom up perspective, there is a diversity of opinions on that question (as indicated by the Credo/Agent Framework JavaScript and Bifold moves), and we’re very interested in the top down perspective. Likewise, the dual-interest relationship between Aries and DIF concerning DIDComm is another aspect of community effort that requires balance and coordination.
```

**Comment by swcurran** [comment]
Update from @TelegramSam applied and notes from Reviewers added.  I hope to add the link from the TOC meeting to the review...

**Review by VRamakrishna — APPROVED** [review]

**Pull request merged**